src/test/java/org/folio/rest/api/InstanceStorageTest.java

```diff
@@ -2708,9 +2708,8 @@ public void cannotCreateInstanceWithDuplicateMatchKey() throws Exception {
     final Response response = createCompleted.get(10, SECONDS);
 
     assertThat(response.getStatusCode(), is(400));
-    assertThat(response.getBody(),
-      is("lower(f_unaccent(jsonb ->> 'matchKey'::text)) value already "
-        + "exists in table instance: match_key"));
+    assertTrue(response.getBody()
+      .contains("Key (lower(f_unaccent(jsonb ->> 'matchKey'::text)))=(match_key) already exists."));
```

Comment:
I believe an old version of this message should also be preserved in case UI depends on it.